packages/plugins/experimental/plugin-script/src/edge/functions.ts

```diff
@@ -37,10 +37,10 @@ export type UploadWorkerProps = {
   credentialLoadTimeout?: number; // ms to wait for credentials to load
 };
 
-const defaultUserFunctionsBaseUrl = 'https://edge-main.dxos.workers.dev'; // 'http://localhost:8600';
+const defaultUserFunctionsBaseUrl = 'https://edge-main.dxos.workers.dev/functions'; // 'http://localhost:8600';
 
 const getBaseUrl = (config: Config) => {
-  return config.get('runtime.app.env.DX_FUNCTIONS_SERVICE_HOST') || defaultUserFunctionsBaseUrl;
+  return config.values.runtime?.services?.edge?.url || defaultUserFunctionsBaseUrl;
```

Comment:
need to append `/functions` to the edge service url.
also make sure its the correct protocol: ws vs http